cmd/minishift/cmd/start.go

```diff
@@ -689,6 +693,34 @@ func initSubscriptionManagerFlags() *flag.FlagSet {
 	return subscriptionManagerFlagSet
 }
 
+// Get config from the start flags and set it to viper config so that in
+// stop start case user don't need to remember the start flags.
+func populateStartFlagsToViperConfig() {
+	startFlagSet.AddFlagSet(clusterUpFlagSet)
+	startFlagSet.VisitAll(func(flag *flag.Flag) {
```

Comment:
@praveenkumar Previously we did not add `clusterUpFlagSet` to ` startFlagSet`, but in this PR we do. So does  it have any side effect?

Reply:
@LalatenduMohanty it shouldn't be we did it because proxy parameter are part of clusterUpFlagSet not StartFlagSet so if we also want to take out that pain from user then this is the best bet we have.